code/game/objects/items/weapons/twohanded.dm

```diff
@@ -236,6 +238,8 @@
 	name = "spear"
 	desc = "A haphazardly-constructed yet still deadly weapon of ancient design."
 	force = 10
+	sharpness = 0.8
+	sharpness_flags = SHARP_TIP | INSULATED_EDGE
```

Comment:
I don't quite like the idea of someone skinning an animal with the tip of their makeshift glass shard spear